frontend/src/pages/interviewlist.js

```diff
@@ -98,15 +98,16 @@ class InterviewListPage extends React.Component<Props> {
                           <h5 style={{ display: 'block' }}>Candidates in {category}</h5>
                         </Row>
                         <div className="m-2">
-                          {this.state.interviewingInterviews.map(interview =>
-                            interviewGetCategorySection(interview) !== null &&
-                            interviewGetCategorySection(interview).response.text === category ? (
-                              <li>
-                                <Link href={`/candidate?id=${interview.candidate_id}`}>
-                                  {interview.candidate_name}
-                                </Link>
-                              </li>
-                            ) : null
+                          {this.state.interviewingInterviews.map(
```

Comment:
hmmmm I noticed that you changed frontend files and now lint is failing. Did you reformat the frontend manually? I think @angadgarg25 suggests that we just leave formatting up to `yarn`, so if you made any changes just change them back. Let me know if my assumption is incorrect though!

Reply:
ohhh hmmm just read your note.. pinging @angadgarg25 to see what's going wrong here

Reply:
try out my comment below